include/boost/astronomy/coordinate/io.hpp

```diff
@@ -24,7 +24,7 @@ template
 >
 std::ostream& operator<< (std::ostream &out, cartesian_representation< CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
 {
-    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_x() << " )";
+    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_z() << " )";
```

Comment:
Break this line to fit in within the limit of 100 characters

Reply:
Sorry about repeatedly missing that rule...Will adhere to it in the future.